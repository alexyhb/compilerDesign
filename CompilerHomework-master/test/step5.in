x=y k="z$var more" cmd -flag=something
