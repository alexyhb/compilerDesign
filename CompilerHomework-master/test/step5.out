Built a parse-tree:
stream:
  optline:has line
    equals:
      WORD:x
      WORD:y
      equals:
        WORD:k
        concatenate:
          DBLQUOTED:z
          VAREXP:var
          DBLQUOTED: more
        command:
          WORD:cmd
          concatenate:
            WORD:-flag
            EQUALS:=
            WORD:something
  optline:empty
